libpolys/polys/simpleideals.cc

```diff
@@ -59,10 +60,10 @@ ideal idInit(int idsize, int rank)
 void idShow(const ideal id, const ring lmRing, const ring tailRing, const int debugPrint)
```

Comment:
No: idShow is a debug routine and must therefore accept id==NULL -> no need to change this routine